infrastructure/src/main/java/org/corfudb/infrastructure/BatchWriter.java

```diff
@@ -162,6 +164,21 @@ public void reset(@Nonnull long epoch) {
         }
     }
 
+    public Long queryLogTail(long epoch) {
```

Comment:
Is it better to return something like TailResponse? All other similar methods returns *Response type.